drivers/sensor/bmm150/bmm150.c

```diff
@@ -645,13 +648,15 @@ static int pm_action(const struct device *dev, enum pm_device_action action)
 			LOG_ERR("failed to enter suspend mode: %d", ret);
 		}
 		break;
+	case PM_DEVICE_ACTION_TURN_OFF:
+		/* Do nothing */
+		break;
```

Comment:
is this really needed?

Reply:
Without this it would fall through to `-ENOTSUP`, which is not strictly true.

Reply:
then it looks like PM subsys needs to handle this scenario, plaguing drivers with this snippet feels wrong.